src/server/common/helpers/fetch/fetch-deployable-image-names.js

```diff
@@ -1,33 +1,19 @@
-import qs from 'qs'
-import { config } from '../../../../config/config.js'
-import { fetchJson } from './fetch-json.js'
-import { getUserGroups } from '../auth/get-user-groups.js'
+import { fetchServices } from './fetch-entities.js'
+import { userIsAdmin } from '../user/user-is-admin.js'
 
 /**
  * @typedef {object} Options
  * @property {import('@hapi/hapi').Request} request
- * @property {string} [scope]
+ * @return [string]
  */
+async function fetchDeployableImageNames({ request }) {
+  const params = {}
+  const userSession = await request.getUserSession()
+  if (!userIsAdmin(userSession)) {
+    params['teamIds'] = userSession.scope
+  }
 
-/**
- * @summary Fetch images a user can deploy
- * @description A user can deploy images they own. They own an image by being a member of the team that owns a
- * service. There are two ways to obtain these deployable images:
- * 1) Preferred: When request is available use this
- * 2) When request is not available pass in user scope/groups
- * @param {Options} options
- * @returns {Promise<*>}
- */
-async function fetchDeployableImageNames({ request, scope }) {
-  const userGroups = scope ?? (await getUserGroups(request))
-
-  const endpoint = `${config.get('portalBackendUrl')}/deployables${qs.stringify(
-    { runMode: 'service', groups: userGroups },
-    { arrayFormat: 'repeat', addQueryPrefix: true }
-  )}`
-
-  const { payload } = await fetchJson(endpoint)
-  return payload
+  return (await fetchServices(params)).map((e) => e.name)
```

Comment:
is this working? the response will come back from `Wreck` on the `payload` object

Reply:
Seems to be working for me locally and in the test suite.
I'll double check but i think fetchServices is already extracting the wreck payload (it returns the full entity list)